src/main/kotlin/com/sourcegraph/cody/ui/CodyToolWindowFactory.kt

```diff
@@ -66,12 +86,40 @@ interface NativeWebviewProvider {
 
 /** A NativeWebviewProvider that thunks to WebUIService. */
 class WebUIServiceWebviewProvider(val project: Project) : NativeWebviewProvider {
+  companion object {
+    private val logger = Logger.getInstance(WebUIServiceWebviewProvider::class.java)
+  }
+
   override fun createPanel(params: WebviewCreateWebviewPanelParams) =
       WebUIService.getInstance(project).createWebviewPanel(params)
 
-  override fun receivedPostMessage(params: WebviewPostMessageStringEncodedParams) =
-      WebUIService.getInstance(project)
-          .postMessageHostToWebview(params.id, params.stringEncodedMessage)
+  override fun receivedPostMessage(params: WebviewPostMessageStringEncodedParams) {
```

Comment:
Can we send a pain message (not string encoded)? If not, what is the proper approach? 

Maybe there is a way to handle copy-to-clipboard wholly in Code? Can we alter `navigator.clipboard.writeText(preText)` from [vscode/webviews/chat/ChatMessageContent/create-buttons.ts](https://sourcegraph.com/github.com/sourcegraph/cody@mkondratek/fix/copy-code/-/blob/vscode/webviews/chat/Chat MessageContent/create-buttons.ts?L173%3A1-222%3A1) to do so?